platform/linux-generic/libodp-linux.pc.in

```diff
@@ -6,7 +6,6 @@ includedir=@includedir@
 Name: lib@ODP_LIB_NAME@
 Description: The ODP packet processing engine
 Version: @PKGCONFIG_VERSION@
-Requires.private: libconfig@AARCH64CRYPTO_PKG@
-Libs: -L${libdir} -l@ODP_LIB_NAME@ @ATOMIC_LIBS_NON_ABI_COMPAT@
-Libs.private: @OPENSSL_STATIC_LIBS@ @DPDK_LIBS@ @PCAP_LIBS@ @PTHREAD_LIBS@ @TIMER_LIBS@ @LIBXDP_LIBS@ -lpthread @ATOMIC_LIBS_ABI_COMPAT@
+Requires: lib@ODP_LIB_NAME@-internal
```

Comment:
Perhaps a few words about how this structure (two separate pkg-config files with `Requires`) accomplishes a correct static linking outcome could be added to the commit log.

Reply:
Added explanation as a comment in the file itself. In principle, the same explanation belongs in odphelper-internal as well, but it's probably enough that it's in one place.